content/kubecarrier/master/tutorials_howtos/how_to_contribute_to_kubecarrier/_index.en.md

```diff
@@ -0,0 +1,63 @@
++++
```

Comment:
Please also add this file to the v0.3 version (the latest stable) - the one that users visit from the https://docs.kubermatic.com/ landinf page.

Reply:
Noted

Reply:
Done